examples/03_connectivity/plot_sphere_based_connectome.py

```diff
@@ -0,0 +1,175 @@
+"""
+Extract signals on spheres from an atlas and plot a connectome
+==============================================================
+
+This example shows how to extract signals from spherical regions
+centered on coordinates from Power-264 atlas [1] and Dosenbach-160 [2].
+We estimate connectome using **sparse inverse covariance**, to recover
+the functional brain networks structure.
+
+References
+----------
+[1] Power, Jonathan D., et al. "Functional network organization of the
+human brain." Neuron 72.4 (2011): 665-678.
+
+[2] Dosenbach N.U., Nardos B., et al. "Prediction of individual brain maturity
+using fMRI.", 2010, Science 329, 1358-1361.
+
+"""
+
+###############################################################################
+# fMRI data and Power atlas loading
+# ---------------------------------
```

Comment:
I think that this title would be more readable if it was written as "Load fMRI data and Power atlas"